apps/web/src/features/agent/useLog.ts

```diff
@@ -4,7 +4,10 @@ import type { ChatMessage } from "window.ai"
 export function useLog() {
```

Comment:
seems like the logic inside of this hook is repeated above in a few places

Reply:
They repeat a little bit but not enough for the useWindowAI hook to adopt this as a dependency yet IMO. Tho, do you think we should start that process?

Reply:
I'm not sure yet, but I'm fine with keeping it as-is, and having useWindowAI be simple and not dependent on things built for demos